h/static/images/icons/logo.svg

```diff
@@ -4,7 +4,9 @@
     <title>Rectangle 2 Copy 17</title>
     <desc>Created with Sketch.</desc>
     <defs></defs>
+    <rect fill="#ffffff" stroke="none" width="17.14407" height="16.046612"
+          x="3.8855932" y="3.9449153" />
```

Comment:
See commit message. I put an opaque white rect behind the "h"-shaped hole so that the "h" always appears white regardless of which background colour the svg file is rendered against.